### PR TITLE
Sql files for fix database

### PR DESCRIPTION
**Pull request by astridx**
Pull Request for Issue # .

### Testing Instructions
1. Make a new installation of Joomla with the branch  4.0-dev in the main repo joomla-cms and see, that the table #__templates_overrides is not created
2. Apply this branch.
3. Go to Installer | Database (index.php?option=com_installer&view=database)
4. Click the toolbar button fix
5. See that the table #__templates_overrides is created.

### CODE REVIEW

File: administrator/components/com_admin/sql/updates/postgresql/4.0.0-2018-07-19.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+	CREATE TABLE IF NOT EXISTS "#__template_overrides" (
```
Comment (Anu1601CS) [inline administrator/components/com_admin/sql/updates/postgresql/4.0.0-2018-07-19.sql]: Delete space

**Comment by Anu1601CS** [comment]
@astridx I am not sure about this. Is this nessecary for all  table ?

@laoneo  ?

File: administrator/components/com_admin/sql/updates/postgresql/4.0.0-2018-07-19.sql (hunk 1 above)
Comment (astridx) [reply administrator/components/com_admin/sql/updates/postgresql/4.0.0-2018-07-19.sql]: Thanks. Done

**Comment by astridx** [comment]
> @astridx  I am not sure about this. Is this nessecary for all table ?

It works without this files. So it is not necessary. But it is a feature. In case something goes wrong you can fix the database. The table is not deleted. We only create in the fix task a table if none exists.
I know this so far only from components. If you create the update-files in the component's sql/update directory, everything will work, even if you skip updates.

In addition, you can also repair the database table of your component.

**Comment by laoneo** [comment]
Yes we need the update scripts too. Thanks.

**Pull request merged**